internal/serve/httphandler/forgot_password_handler_test.go

```diff
@@ -57,51 +72,77 @@ func Test_ForgotPasswordHandler(t *testing.T) {
 		SDPUIBaseURL: &uiBaseURL,
 	}
 	ctx := tenant.SaveTenantInContext(context.Background(), &tnt)
+	const email = "valid@email.com"
+	defaultUserRoles := []string{data.OwnerUserRole.String()}
```

Comment:
When I read `defaultUserRoles` throughout this file, I need to come back often to its definition to understand which role is the default. We should use a name that conveys the meaning of the value. What do you think about calling it `ownerUserRoleSlice`?

Or maybe just passing the const data.OwnerUserRole?

If your goal here is to avoid instantiating the []string slice every time, you could use [variadic functions](https://gobyexample.com/variadic-functions).

Reply:
good idea, I think the naming I introduced is not too intuitive for someone touching this test file for the first time